### DISCUSSION TITLE
Future SDKs

### DISCUSSION THREAD

**Discussion by markphelps** [flipt-io/flipt#2208] [Polls]
This poll will be used to gauge interest in additional clients/SDKs for Flipt.

Please upvote languages you'd like us to support.

If you'd like a language (or protocol) that's not on the poll, please comment.

**Comment by alexcardell** [comment]
Scala (ideally with a Typelevel wrapper)

We're trying out Flipt at $work so will likely have to write something that could be a candidate

**Reply by markphelps** [reply]
@alexcardell Scala would be awesome! Unfortunately I don't think anyone on the team is very familiar with it, so could use some help there I think

**Reply by alexcardell** [reply]
I did start a little feature flag library intended to use pluggable backends, but haven't spent much time on it. Planning to look into the OpenFeature spec and see if I can implement a more idiomatic FP Scala client for it.

**Comment by markphelps** [comment]
FYI we now have a PHP REST client in beta https://github.com/flipt-io/flipt-php

Please give it a try and file any issues you may have

**Comment by piclemx** [comment]
Openfeature Javascript

**Reply by markphelps** [reply]
On it!

**Reply by markphelps** [reply]
We may need some help making it idiomatic though as we have limited node expertise on our core team

**Reply by piclemx** [reply]
That's fine. 😄 We can help and contribute if needed :)

**Reply by markphelps** [reply]
@piclemx we will start this next week and keep you updated!

**Reply by piclemx** [reply]
@markphelps Any news?

**Reply by markphelps** [reply]
@piclemx yup! we have a WIP PR here: https://github.com/open-feature/js-sdk-contrib/pull/724 

@yquansah is just working on adding tests before opening it for the open feature team to review

**Reply by markphelps** [reply]
@piclemx https://github.com/open-feature/js-sdk-contrib/pull/724 was just merged thanks to @yquansah and @beeme1mr !! just waiting on the release to NPM to happen now

**Reply by markphelps** [reply]
@piclemx it's alive! https://www.npmjs.com/package/@openfeature/flipt-provider

**Comment by markphelps** [comment]
I'm going to close this because I realize a poll is likely not the best way to capture this as I didn't realize you couldn't add to a poll after creating it.. makes sense in retrospect.

I think a better avenue for requesting SDKs would be opening issues in either of our new [server-side evaluation SDKs](https://github.com/flipt-io/flipt-server-sdks) or [client-side evaluation SDKs](https://github.com/flipt-io/flipt-client-sdks) for any language you'd like to see. Then others can upvote etc.

I'll create issues for C# (server), Scala (server), and Ruby (server). For Swift, I'm not sure if server side or client side would be more useful, so hopefully the OP can open an issue in the respective repo.

Thanks all!! 🙏🏻

**Comment by anthony-cassaigne** [comment]
Openfeature python will be perfect.

**Reply by markphelps** [reply]
@anthony-cassaigne 💯 agree, that would round out our Python support quite nicely.

Any interest/availability in helping with this?

**Reply by lukas-reining** [reply]
OpenFeature python demand is here from us too @markphelps.
I am not sure when I would get to help implementing this.

**Comment by falghi** [comment]
OpenFeature Ruby would be nice

**Comment by vpo-cdc** [comment]
OpenFeature PHP would be awesome
